src/components/changePlanner/CPDetailView.tsx

```diff
@@ -358,157 +392,196 @@ class CPDetailView extends React.Component<
     }
     return (
       <div className={Classes.DARK + " asset-view"}>
+        <Alert
+          cancelButtonText="Cancel"
+          confirmButtonText="Execute"
+          intent={Intent.WARNING}
+          className={Classes.DARK}
+          isOpen={this.state.isAlertOpen}
+          onCancel={() => this.handleExecuteCancel()}
+          onConfirm={() => this.handleExecute()}
```

Comment:
after confirming execute, the execute & work order buttons do not automatically switch to be disabled unless you refresh the page

Reply:
fixed